demos/pet-demo/src/components/HomeRoute.tsx

```diff
@@ -6,8 +6,8 @@ import { ErrorBoundary } from './ErrorBoundary';
 import { FullPageLoading } from './routes';
 
 export const HomeRoute = iso(`
-  field Query.HomeRoute @component {
-    pets {
+  field Query.HomeRoute($search: String, $statsWhere: PetStatsWhere) @component {
```

Comment:
I use `HomeRoute` and `DetailRoute` when presenting about Isograph. Can we add this to some other entrypoint instead so that demos still look clean?